src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,69 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.sql.SQLException;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+import mate.academy.util.InitDatabase;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy.dao");
+
     public static void main(String[] args) {
+        try {
+            InitDatabase.executeInitScript(ConnectionUtil.getConnection());
```

Comment:
It's a good practice to close the Connection object after you're done with it to avoid potential memory leaks. Consider using try-with-resources to ensure that the Connection is closed properly.